src/lib/controller.ts

```diff
@@ -38,6 +41,9 @@ export class AngusController {
 
     async updateModel(model: AngusModel) {
         let key: string = this.createCompositeKey(this.name, model.getSplitKey());
+        let oldModel: AngusModel = await this.getModel(key);
```

Comment:
This request will trigger a 2nd evalueate CC/db call for the same model. e.q.: we already have the model in our actual controller :eyes https://github.com/arvenis/peertix-developer-stack/blob/75d9cbcff674631cc700f3afab69b0dbbba1145c/libs/angus-chaincode/chaincode-contracts/src/lib/event/event.contract.ts#L96

So I am not sure this is the best way to NOT lose the createdAt timestamp for a model during update.